pkg/minishift/openshift/openshift.go

```diff
@@ -44,8 +43,8 @@ var (
 
 	NODE = OpenShiftPatchTarget{
 		"node",
-		"/var/lib/origin/openshift.local.config/node-%s/node-config.yaml",
-		"/var/lib/minishift/openshift.local.config/node-%s/node-config.yaml",
+		"/var/lib/origin/openshift.local.config/node-localhost/node-config.yaml",
```

Comment:
ahh, and that will again work backwards compatible, right? Since we are using 3.6 now to provision say 1.5.